src/sse/route-service.ts

```diff
@@ -115,7 +125,7 @@ export default class RouteService {
 }
 
 function parseParam(value, defaultValue = 0) {
-  if (!value) {
+  if (value === undefined) {
```

Comment:
We should probably add types for `value` here and for the returntype of the function.

_Note: this goes hand-in-hand with the comment about the linter config from @kadena-dev/eslint-config_